tests/test_vox.py

```diff
@@ -95,12 +95,12 @@ def test_activate_non_vox_venv(xonsh_builtins, tmpdir):
     @xonsh_builtins.events.vox_on_activate
     def activate(name, **_):
         nonlocal last_event
-        last_event = "activate", name
+        last_event = "activate", name, _['path']
```

Comment:
... you could just add it to the signature?

Reply:
Yes! I hadn't finished with my implementation yet and I still wanted to add the documentation for the new parameter.  Anyhow, that is now done so hopefully we're good to merge.